src/sentry/api/endpoints/debug_files.py

```diff
@@ -443,10 +443,16 @@ def serialize_results(results):
                 [expose_release(r, file_count_map.get(r["id"], 0)) for r in results], request.user
             )
 
+        sort_by = request.GET.get("sortBy", "-date_added")
+        if sort_by not in {"-date_added", "date_added"}:
```

Comment:
```
if sort_by.lstrip("-") != "date_added":
```

Reply:
YOu can see it works well with both:
```
>>> sort_by = "date_added"
>>> sort_by.lstrip("-")
'date_added'
>>>
```

Reply:
I don't like having to hardcore such logic for a such simple thing. I would prefer to be explicit.

Reply:
I also feel that way, as we agreed we could follow up with the serializers and all the proper implementations.